ur_controllers/src/scaled_joint_trajectory_controller.cpp

```diff
@@ -262,6 +267,7 @@ controller_interface::return_type ScaledJointTrajectoryController::update(const
               result->set__error_code(FollowJTrajAction::Result::GOAL_TOLERANCE_VIOLATED);
               active_goal->setAborted(result);
               rt_active_goal_.writeFromNonRT(RealtimeGoalHandlePtr());
+
```

Comment:
Would we need to also set `traj_point_active_ptr_ = nullptr;` at this line, since we reset the goal pointer here?

Reply:
yes i thought about that too, but i did not see it in the original too when return error code goal_tolerance_violated.
this could be an error on the original or not needed or potentially breaking if inserted.
Furthermore it set's the nullptr on_cleanup and on_reset methods which i think are inhereted into the SJTC